src/kernel/kshell.c

```diff
@@ -294,6 +294,12 @@ int try_exec(const char* command) {
     return 0;
 }
 
+int overflowtest() {
```

Comment:
Can we rename this function to match the command name then? (or we rename the command, I don't mind).

Reply:
Done